packages/server/lib/webhook/github-app-oauth-webhook-routing.ts

```diff
@@ -90,7 +90,7 @@ async function handleCreateWebhook(nango: InternalNango, body: any): Promise<Res
         const [connection] = connections;
 
         // if there is no matching connection or if the connection config already has an installation_id, exit
-        if (!connection || connection.connection_config['installation_id']) {
+        if (!connection || (connection.connection_config['installation_id'] !== undefined && connection.connection_config['installation_id'] !== null)) {
```

Comment:
can installation_id be false or 0? if not I don't think it is worth the extra condition